src/main/scala/org/broadinstitute/dsde/workbench/avram/Avram.scala

```diff
@@ -32,14 +35,15 @@ import org.broadinstitute.dsde.workbench.avram.util.SlickDatabaseFactory
   * back to us and might be worth exploring.
   */
 object Avram {
-  private val databaseFactory = new SlickDatabaseFactory(AvramConfig.dbcpDataSourceConfig)
 
-  val database = databaseFactory.database
+  val database = DbReference(AvramConfig.dbcpDataSourceConfig)
+
   val samDao: SamDao = new HttpSamDao(AvramConfig.sam.baseUrl)
 
   /**
     * DBCP data source provided only for introspection into the database pool statistics. If you're
     * not working with database monitoring, don't use this!
     */
-  val dbcpDataSource: BasicDataSource = databaseFactory.dbcpDataSource
-}
\ No newline at end of file
+  val dbcpDataSource: BasicDataSource = database.dbcpDataSource
```

Comment:
Let's get rid of this since `AvramRoutes.dbPoolStats` can just access it using `Avram.database.dbcpDataSource`. I don't know why I added this here.